certify-service/src/main/java/io/mosip/certify/advice/ExceptionHandlerAdvice.java

```diff
@@ -130,6 +131,9 @@ private ResponseEntity<ResponseWrapper> handleInternalControllerException(Except
             String errorCode = ((CertifyException) ex).getErrorCode();
             return new ResponseEntity<ResponseWrapper>(getResponseWrapper(errorCode, getMessage(errorCode)), HttpStatus.OK);
         }
+        if(ex instanceof TemplateException) {
+            return new ResponseEntity<>(getResponseWrapper(INVALID_REQUEST, ex.getMessage()) ,HttpStatus.NOT_FOUND);
+        }
```

Comment:
When svg-template with a given id is not in table, then in that case response was proper as "Template not found" but the status code was 200. So this response was set so that the status code is also 400 in that case.

Reply:
Rename TemplateException to RenderingTemplateException
Use certifyException for the velocity template code.